pkg/apps/blackduck/latest/deployer.go

```diff
@@ -52,7 +51,7 @@ func (hc *Creater) getPostgresComponents(blackduck *blackduckapi.Blackduck) (*ap
 		adminPassword = blackduck.Spec.ExternalPostgres.PostgresAdminPassword
 		userPassword = blackduck.Spec.ExternalPostgres.PostgresUserPassword
 	} else {
-		adminPassword, userPassword, _, err = bdutil.GetDefaultPasswords(hc.KubeClient, hc.Config.Namespace)
+		adminPassword, userPassword, _, _, err = util.GetDefaultPasswords(hc.KubeClient, hc.Config.Namespace)
```

Comment:
Probably you need to fix the merge conflict of this PR with Yash changes related to password